### PR TITLE
fix: ui regression on index.mdx

### PR DESCRIPTION
**Pull request by cpcramer**
**What changed? Why?**
Fix UI regression in index.mdx that was introduced through this [PR](https://github.com/coinbase/onchainkit/pull/659). 

**Notes to reviewers**

**How has it been tested?**

### CODE REVIEW

File: site/docs/pages/index.mdx [hunk 1]
```diff
@@ -84,14 +84,14 @@ bun add @coinbase/onchainkit
       <div>
 
 ```tsx
-<OnchainKitProvider
-  chain={base}
-  schemaId="0xf8b05c79f090979bf4a80270aba232dff11a10d9ca55c4f88de95317970f0de9"
->
+<OnchainKitProvider // [!code focus]
```
Comment (Zizzamia) [inline site/docs/pages/index.mdx:87]: Open question, why you think before we didn't have `code focus` here?

File: site/docs/pages/index.mdx [hunk 2]
```diff
@@ -140,7 +140,7 @@ bun add @coinbase/onchainkit
     return (
       <div className="flex h-8 w-8 items-center justify-center">
         <button type="button" onClick={() => disconnect()}>
-          <Avatar address={address} />
+          <Avatar address={address} /> // [!code focus]
```
Comment (Zizzamia) [inline site/docs/pages/index.mdx:143]: Same, here why before we didn't have `code focus` here?

File: site/docs/pages/index.mdx (hunk 1 above)
Comment (cpcramer) [reply site/docs/pages/index.mdx:87]: I assumed we need to highlight any onchainKit components. Should I remove it here?

Comment (Zizzamia) [reply site/docs/pages/index.mdx]: So let me ask you a question, why you think it was not highlighted before?

**Review by Zizzamia — APPROVED** [review]

**Pull request merged**